cmd/apikey_save.go

```diff
@@ -1,26 +1,51 @@
 package cmd
 
 import (
+	"bufio"
 	"fmt"
 	"os"
 
 	"github.com/civo/civogo"
 	"github.com/civo/cli/config"
 	"github.com/civo/cli/utility"
 	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
 )
 
 var apikeySaveCmd = &cobra.Command{
 	Use:     "save",
 	Aliases: []string{"add", "store", "create", "new"},
 	Short:   "Save a new API key",
-	Args:    cobra.MinimumNArgs(2),
+	Args:    cobra.MinimumNArgs(0),
 	Example: "civo apikey save NAME KEY",
 	Run: func(cmd *cobra.Command, args []string) {
-		config.Current.APIKeys[args[0]] = args[1]
 
+		var username, apikey string
+		var err error
+		if len(args) == 0 {
+			reader := bufio.NewReader(os.Stdin)
+			fmt.Printf("Enter Username: ")
+
+			username, err = reader.ReadString('\n')
+			if err != nil {
+				utility.Error("Error reading username")
+				os.Exit(1)
+			}
+			fmt.Printf("Now, please type in the apikey: ")
```

Comment:
"Enter the API key:"

(no need for "Now, ", keeping the language consistent with the other prompt - you used "enter" not "type", and it was in an instructional format rather than request, i.e. no "please")